packages/roles/src/Roles.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * A user roles class for Jetpack.
+ *
+ * @package jetpack-roles
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class Automattic\Jetpack\Roles
+ *
+ * Contains utilities for translating user roles to capabilities and vice versa.
+ */
+class Roles {
+	/**
+	 * Map of roles we care about, and their corresponding minimum capabilities.
+	 *
+	 * @access protected
+	 * @static
+	 *
+	 * @var array
+	 */
+	protected static $capability_translations = array(
```

Comment:
do we want this static?

Reply:
Good question - maybe not. We won't be using it outside, so I'll make it non-static.

Reply:
Done in 3a9c8f3f90ed64420057901dab25ea377c51d5b8